src/test/java/redis/clients/jedis/modules/search/SearchDefaultDialectTest.java

```diff
@@ -1,19 +1,27 @@
 package redis.clients.jedis.modules.search;
 
-import static org.junit.Assert.*;
+import static org.hamcrest.MatcherAssert.assertThat;
+import static org.hamcrest.Matchers.containsString;
+import static org.hamcrest.Matchers.emptyString;
```

Comment:
All `emptyString()` -> `emptyOrNullString()`.
```suggestion
import static org.hamcrest.Matchers.emptyOrNullString;
```